decidim-admin/app/controllers/decidim/admin/participatory_processes_controller.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+require_dependency "decidim/admin/application_controller"
+
+module Decidim
+  module Admin
+    # Controller that allows managing all the Admins.
+    #
+    class ParticipatoryProcessesController < ApplicationController
+      def index
+        @participatory_processes = collection
+        authorize @participatory_processes
+      end
+
+      def new
+        @form = ParticipatoryProcessForm.new
+        authorize ParticipatoryProcess
+      end
+
+      def create
+        @form = ParticipatoryProcessForm.from_params(params)
+        authorize ParticipatoryProcess
+
+        CreateParticipatoryProcess.call(@form, current_organization) do
+          on(:ok) do
+            flash[:notice] = I18n.t("participatory_processes.create.success", scope: "decidim.admin")
+            redirect_to participatory_processes_path
+          end
+
+          on(:invalid) do
+            flash.now[:alert] = I18n.t("participatory_processes.create.error", scope: "decidim.admin")
+            render :new
+          end
+        end
+      end
+
+      def edit
+        @participatory_process = collection.find(params[:id])
+        authorize @participatory_process
+        @form = ParticipatoryProcessForm.from_model(@participatory_process)
+      end
+
+      def update
+        @participatory_process = collection.find(params[:id])
+        authorize @participatory_process
+        @form = ParticipatoryProcessForm.from_params(params)
+
+        UpdateParticipatoryProcess.call(@participatory_process, @form) do
```

Comment:
Then we'll need to customize the DSL for that case when we need to do something in between, and we'll end up having `inherited_resources`